moneo.py

```diff
@@ -146,9 +146,15 @@ def deploy_worker(self, hosts_file, max_threads=16):
             cmd = cmd + ' false'
         if self.args.launch_publisher:
             agent = self.args.launch_publisher
-            print('-Enable ' + agent + ' agent-')
-            logging.info('Enable ' + agent + ' agent')
             cmd = cmd + ' ' + agent
+            if self.args.publisher_auth:
```

Comment:
Whats the functionality if a user selects to use the publisher and doesn't provide the auth argument?

Reply:
Thanks for catching, addressed.